### PR TITLE
Mycus and Folken update (Finished)

### PR DESCRIPTION
**Pull request by Kazkin**
-Added mycus and folken as playable races.
-Added hair styles for mycus and folken.
-Added some unique plant organs.
-Added some unique coloring and naming to slime-mold/shroomling companions.
-Made healing animal companions much much much better mathmatically.

## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: code/datums/perks/racial.dm [hunk 1]
```diff
@@ -512,23 +512,26 @@
...
 	name = "Mycus Regeneration"
-	desc = "As a mycus, you heal as long as you are in the darkness."
+	desc = "As a mycus, you heal as long as you are in the darkness, increasing your natural regeneration."
 	passivePerk = TRUE
 
 /datum/perk/mushroom_follower
-	name = "Spawn Mushroom Follower"
-	desc = "Mushroom followers produces random healing chems when fed enough food."
+	name = "Spawn Shroomling"
+	desc = "Shroomlings are animal-intelligence mycus capable of following simple orders like 'Shroomling 'Name' Follow.' and 'Shroomling 'Name' Stop.' who will stay by you when ordered. While capable of fighting, they are quite weak, the \
```
Comment (r4d6) [inline code/datums/perks/racial.dm:533]: I think it should mention that you need to mention their name & follow/stop in the same sentence.

File: code/datums/perks/racial.dm [hunk 2]
```diff
@@ -548,8 +551,9 @@
 	..()
 
 /datum/perk/slime_follower
-	name = "Spawn Slime-Mold Follower"
-	desc = "Slime followers regenerates and has better stats fit for combat."
+	name = "Spawn Slime-Mold"
+	desc = "Slime-mold shroomlings are animal-intelligence mycus capable of following simple orders like 'Slime-Mold 'Name' Follow.' and 'Slimd-Mold 'Name' Stop.' who will stay by you when ordered. Slime-molds are made for combat, being \
```
Comment (r4d6) [inline code/datums/perks/racial.dm:555]: Same here

File: code/datums/setup_option/backgrounds/origin_homeworld.dm [hunk 3]
```diff
@@ -1,3 +1,25 @@
+/datum/category_item/setup_option/background/homeworld
+	restricted_to_species = list(FORM_HUMAN, FORM_SABLEKYNE, FORM_AKULA, FORM_MARQUA, FORM_NARAMAD, FORM_OPIFEX, FORM_CHTMANT, FORM_KRIOSAN, FORM_CINDAR, FORM_SOTSYNTH, FORM_AGSYNTH, FORM_BSSYNTH, FORM_CHURCHSYNTH)
+
+/datum/category_item/setup_option/background/homeworld/amereth
+	name = "Amereth"
+	desc = "The great jewel of the frontier, Amereth, the place you have lived formerly as a tribal (or perhaps currently) that contains all the wonder and mystery of the realm. It's \
```
Comment (r4d6) [inline code/datums/setup_option/backgrounds/origin_homeworld.dm:6]:
```suggestion
	name = "Amethyn"
	desc = "The great jewel of the frontier, Amethyn, the place you have lived formerly as a tribal (or perhaps currently) that contains all the wonder and mystery of the realm. It's \
```

File: code/datums/setup_option/backgrounds/origin_homeworld.dm [hunk 4]
```diff
@@ -1,3 +1,25 @@
+/datum/category_item/setup_option/background/homeworld
+	restricted_to_species = list(FORM_HUMAN, FORM_SABLEKYNE, FORM_AKULA, FORM_MARQUA, FORM_NARAMAD, FORM_OPIFEX, FORM_CHTMANT, FORM_KRIOSAN, FORM_CINDAR, FORM_SOTSYNTH, FORM_AGSYNTH, FORM_BSSYNTH, FORM_CHURCHSYNTH)
+
+/datum/category_item/setup_option/background/homeworld/amereth
+	name = "Amereth"
```
Comment (r4d6) [inline code/datums/setup_option/backgrounds/origin_homeworld.dm:5]:
```suggestion
	name = "Amethyn"
```

File: code/datums/setup_option/backgrounds/origin_homeworld.dm (hunk 3 above)
Comment (r4d6) [inline code/datums/setup_option/backgrounds/origin_homeworld.dm:6]:
```suggestion
	desc = "The great jewel of the frontier, Amethyn, the place you have lived formerly as a tribal (or perhaps currently) that contains all the wonder and mystery of the realm. It's \
```

File: code/modules/mob/living/carbon/superior_animal/lodge/types/cerberus.dm [hunk 5]
```diff
@@ -3,7 +3,7 @@
 	name = "cerberus"
 	desc = "A domesticated hell diver kept fat, happy, and loyal by the local hunting lodge that breed them as hunting allies and guard animals. Favored especially for their asexual \
 	reproduction after being fed various mushrooms, a fact that baffles some soteria scientists. Unlike a standard hell diver the cerberus isn't as \
-	obsessed with food but gladly lets itself get pulled along. The instinct to attack insectile creatures that makes these attempt to maul cht'mants to death, has been bred out of them."
+	obsessed with food but gladly lets itself get pulled along. Special training allows for them to follow anyone who states 'Cerberus Follow', while 'Cerberus Stop' will make them halt."
```
Comment (r4d6) [inline code/modules/mob/living/carbon/superior_animal/lodge/types/cerberus.dm:6]: Again, now it just need to have the creature's name & Follow/Stop in the same sentence

File: code/modules/mob/living/carbon/superior_animal/lodge/types/cerberus.dm [hunk 6]
```diff
@@ -46,7 +47,7 @@
 /mob/living/carbon/superior_animal/lodge/cerberus/chimera
 	name = "chimera"
 	desc = "A mutated strain of a domesticated cerberus, rarely appearing and much sleeker than their lesser cousins. Chimera are faster, better armored, and much more lethal than a cerberus \
-	thanks to its maw containing a huge singular fang, making them prized companions by all hunters. The instinct to attack insectile creatures that makes these attempt to maul cht'mants to death, has been bred out of them."
+	thanks to its maw containing a huge singular fang, making them prized companions by all hunters. Special training allows for them to follow anyone who states 'Chimera Follow', while 'Chimera Stop' will make them halt."
```
Comment (r4d6) [inline code/modules/mob/living/carbon/superior_animal/lodge/types/cerberus.dm:50]: Ditto

File: code/datums/perks/racial.dm (hunk 1 above)
Comment (Kazkin) [reply code/datums/perks/racial.dm:533]: 'Shroomling 'Name' Follow.' is in the perk name.

Comment (r4d6) [reply code/datums/perks/racial.dm:533]: I meant that, from the way it is written, it seems at if it **must** follow that format, when you just need to mention their names (which include the 'Shroomling' in front, I don't know why you did that) and 'follow' or 'stop' in the same sentence.

**Pull request merged**